core/src/test/java/kafka/test/ClusterInstance.java

```diff
@@ -159,9 +158,7 @@ default Set<GroupProtocol> supportedGroupProtocols() {
         Set<GroupProtocol> supportedGroupProtocols = new HashSet<>();
         supportedGroupProtocols.add(CLASSIC);
 
-        // KafkaConfig#isNewGroupCoordinatorEnabled check both NEW_GROUP_COORDINATOR_ENABLE_CONFIG and GROUP_COORDINATOR_REBALANCE_PROTOCOLS_CONFIG
-        if (serverProperties.getOrDefault(NEW_GROUP_COORDINATOR_ENABLE_CONFIG, "").equals("true") ||
```

Comment:
do we plan to remove this config value from GroupCoordinatorConfig? I see it was removed from a lot of files, but there are still a few where it is used.

Reply:
Mostly confused because we don't check for the config in kafka apis anymore.

Reply:
We will eventually remove it for sure. This (private) config is still the one use to determine whether the new group coordinator should be enabled or not. The trick to know is that it is automatically set to true when `consumer` is specified as a protocol. We made this to simplify the early access and the preview.

The handling was a bit messy before this patch as the new protocol was enabled whenever the new coordinator ran. It is better now.

Reply:
Hmmm -- do when you say the new group coordinator -- you mean the code and not the protocol.

Do I have this right?
1. If the protocol is consumer -> new coordinator
2. If the config is enabled -> new coordinator
3. If protocol is not consumer && config is not enabled  -> old coordinator

Reply:
Correct!